Update PyCharm base image to bookworm

> Most of the new Hs are Linux kernel. The PyCharm image in which most of these occur is [based](https://github.com/DataBiosphere/azul-docker-pycharm/blob/93e58560e4565b2e04781580dee18df6b5da9b2c/.github/workflows/docker-publish.yml#L24) on bullseye which has been [superseded](https://www.debian.org/releases/bullseye/) by bookworm. When we update to PyCharm 3.12 we should also update to the `-slim-bookworm` image, from the `-slim-bullseye` one.

_Originally posted by @hannes-ucsc in https://github.com/DataBiosphere/azul/issues/6537#issuecomment-2354397961_

The following updates are compatible with Azul, this updates the [azul-docker-pycharm](https://github.com/DataBiosphere/azul-docker-pycharm) base image to use bookworm:

```diff
Index: .github/workflows/docker-publish.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.github/workflows/docker-publish.yml b/.github/workflows/docker-publish.yml
--- a/.github/workflows/docker-publish.yml	(revision e3f0ebc534c36385c058b236d80aa97a56a74ee2)
+++ b/.github/workflows/docker-publish.yml	(date 1726609551020)
@@ -21,7 +21,7 @@
   # value isn't reflected in the tag for this image, you also need to bump the
   # internal version below when changing this variable.
   #
-  azul_docker_pycharm_base_image_tag: bullseye-20240904
+  azul_docker_pycharm_base_image_tag: bookworm-20240904
 
   # The version of PyCharm to install in this image.
   #
@@ -31,7 +31,7 @@
   # upstream image followed by the value of this variable. Increment this value
   # to update the OS packages installed in the image.
   #
-  azul_docker_pycharm_internal_version: 33
+  azul_docker_pycharm_internal_version: 34
 
 jobs:
   build:

```


<details><summary> Tested successfully by running: `azul_docker_registry="" make format`</summary>
 

```
azul_docker_registry="" make format
docker run \
	    --rm \
	    --volume $(python scripts/resolve_container_path.py /Users/achave11/Pycharm/Azul/azul.sc):/home/developer/azul \
	    --workdir /home/developer/azul \
	    $(AZUL_DEBUG=0 python -m azul 'docker.resolve_docker_image_for_launch("pycharm")') \
	    /opt/pycharm/bin/format.sh -r -settings .pycharm.style.xml -mask '*.py' src scripts test lambdas/layer/app.py lambdas/indexer/app.py lambdas/service/app.py .flake8/azul_flake8.py environment.py deployments/abrahamsc.local/environment.py deployments/achave.local/environment.py deployments/anvilbox/environment.py deployments/anvildev.browser/environment.py deployments/anvildev.gitlab/environment.py deployments/anvildev.shared/environment.py deployments/anvildev/environment.py deployments/anvilprod.browser/environment.py deployments/anvilprod.gitlab/environment.py deployments/anvilprod.shared/environment.py deployments/anvilprod/environment.py deployments/dev.browser/environment.py deployments/dev.gitlab/environment.py deployments/dev.shared/environment.py deployments/dev/environment.py deployments/gitlab.local/environment.py deployments/hammerbox/environment.py deployments/prod.gitlab/environment.py deployments/prod.shared/environment.py deployments/prod/environment.py deployments/sandbox/environment.py deployments/tempdev.browser/environment.py deployments/tempdev.gitlab/environment.py deployments/tempdev.shared/environment.py deployments/tempdev/environment.py .github/workflows/schedule.py terraform/backend.tf.json.template.py terraform/common.tf.json.template.py terraform/step_function.tf.json.template.py terraform/lambda_layer.tf.json.template.py terraform/dynamo.tf.json.template.py terraform/cloudwatch.tf.json.template.py terraform/elasticsearch.tf.json.template.py terraform/api_gateway.tf.json.template.py terraform/providers.tf.json.template.py terraform/authentication.tf.json.template.py terraform/sqs.tf.json.template.py terraform/s3.tf.json.template.py terraform/gitlab/gitlab.tf.json.template.py terraform/shared/shared.tf.json.template.py terraform/browser/browser.tf.json.template.py lambdas/indexer/.chalice/config.json.template.py lambdas/service/.chalice/config.json.template.py .github/pull_request_template.md.template.py
envhook.py: Warning: azul_docker_registry is '' but should be '122796619775.dkr.ecr.us-east-1.amazonaws.com/', you must run `source environment`
envhook.py: Warning: AZUL_DEBUG is '0' but should be '1', you must run `source environment`
envhook.py: Warning: azul_docker_registry is '' but should be '122796619775.dkr.ecr.us-east-1.amazonaws.com/', you must run `source environment`
2024-09-17 15:01:52,099    INFO MainThread azul.docker: Resolving 'pycharm' image TagImageRef(registry='localhost:5000', username='docker.io', repository=('ucscgi', 'azul-pycharm'), tag='2024.2.1-34') …
2024-09-17 15:01:52,100    INFO MainThread azul.docker: Resolved 'pycharm' image to DigestImageRef(registry='localhost:5000', username='docker.io', repository=('ucscgi', 'azul-pycharm'), digest='sha256:e31a93d76cddfcc147ffc8c3954660e02cbc2c3b8798490d0017dacd2f27c8f2')
2024-09-17 15:01:52,100    INFO MainThread azul.docker: Pulling image DigestImageRef(registry='localhost:5000', username='docker.io', repository=('ucscgi', 'azul-pycharm'), digest='sha256:e31a93d76cddfcc147ffc8c3954660e02cbc2c3b8798490d0017dacd2f27c8f2') …
2024-09-17 15:01:52,154    INFO MainThread azul.docker: Pulled image DigestImageRef(registry='localhost:5000', username='docker.io', repository=('ucscgi', 'azul-pycharm'), digest='sha256:e31a93d76cddfcc147ffc8c3954660e02cbc2c3b8798490d0017dacd2f27c8f2')
Sep 17, 2024 10:01:52 PM java.util.prefs.FileSystemPreferences$1 run
INFO: Created user preferences directory.
PyCharm 2024.2.1, build PC-242.21829.153 Formatter
2024-09-17 22:01:53,118 [    535] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - org.jetbrains.plugins.textmate.TextMateFileType PluginClassLoader(plugin=PluginDescriptor(name=TextMate Bundles, id=org.jetbrains.plugins.textmate, descriptorPath=plugin.xml, path=/opt/pycharm/plugins/textmate, version=242.21829.153, package=null, isBundled=true), packagePrefix=null, state=active, parents=, )
java.lang.ClassNotFoundException: org.jetbrains.plugins.textmate.TextMateFileType PluginClassLoader(plugin=PluginDescriptor(name=TextMate Bundles, id=org.jetbrains.plugins.textmate, descriptorPath=plugin.xml, path=/opt/pycharm/plugins/textmate, version=242.21829.153, package=null, isBundled=true), packagePrefix=null, state=active, parents=, )
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass(ComponentManagerImpl.kt:1464)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass$default(ComponentManagerImpl.kt:1460)
	at com.intellij.serviceContainer.ComponentManagerImpl.loadClass(ComponentManagerImpl.kt:887)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.instantiateFileTypeBean(FileTypeManagerImpl.java:519)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.mergeOrInstantiateFileTypeBean(FileTypeManagerImpl.java:483)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.instantiatePendingFileTypes(FileTypeManagerImpl.java:476)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.initializeComponent(FileTypeManagerImpl.java:653)
	at com.intellij.configurationStore.ComponentStoreImpl.initComponent(ComponentStoreImpl.kt:172)
	at com.intellij.configurationStore.ComponentStoreWithExtraComponents.initComponent(ComponentStoreWithExtraComponents.kt:46)
	at com.intellij.serviceContainer.ComponentManagerImpl.initializeService$intellij_platform_serviceContainer(ComponentManagerImpl.kt:651)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceInitializer.kt:70)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance(ServiceInstanceInitializer.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invokeSuspend(LazyInstanceHolder.kt:162)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:163)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invokeSuspend(LazyInstanceHolder.kt:160)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startCoroutineUndispatched(Undispatched.kt:27)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:90)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:123)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:52)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.initialize(LazyInstanceHolder.kt:145)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.access$initialize(LazyInstanceHolder.kt:13)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.tryInitialize(LazyInstanceHolder.kt:135)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance(LazyInstanceHolder.kt:95)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance$suspendImpl(LazyInstanceHolder.kt:83)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.InstanceContainerImpl.instance(InstanceContainerImpl.kt:68)
	at com.intellij.serviceContainer.ComponentManagerImpl.getServiceAsync(ComponentManagerImpl.kt:694)
	at com.intellij.platform.ide.bootstrap.AppServicePreloadingKt$postAppRegistered$1$fileTypeManagerJob$1.invokeSuspend(appServicePreloading.kt:145)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
2024-09-17 22:01:53,122 [    539] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - PyCharm 2024.2.1  Build #PC-242.21829.153
2024-09-17 22:01:53,122 [    539] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - JDK: 21.0.3; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-09-17 22:01:53,122 [    539] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - OS: Linux
2024-09-17 22:01:53,123 [    540] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - Last Action:
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.users.BackendUserManager, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.ports.CwmPortForwardingToolWindowManager, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.users.BackendCWMUserFocusHost, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.following.FollowMeManagerService, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.following.BackendProjectViewSync, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.following.BackendFollowMeEditorsHost, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.following.BackendFollowMeDebuggerHost, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,437 [    854]   WARN - #c.i.s.ComponentManagerImpl - `preload=TRUE` must be used only for core services (service=com.intellij.cwm.plugin.following.FollowMeLookupSynchronizer, plugin=com.jetbrains.codeWithMe)
2024-09-17 22:01:53,627 [   1044] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - org.jetbrains.plugins.textmate.TextMateFileType PluginClassLoader(plugin=PluginDescriptor(name=TextMate Bundles, id=org.jetbrains.plugins.textmate, descriptorPath=plugin.xml, path=/opt/pycharm/plugins/textmate, version=242.21829.153, package=null, isBundled=true), packagePrefix=null, state=active, parents=, )
java.lang.ClassNotFoundException: org.jetbrains.plugins.textmate.TextMateFileType PluginClassLoader(plugin=PluginDescriptor(name=TextMate Bundles, id=org.jetbrains.plugins.textmate, descriptorPath=plugin.xml, path=/opt/pycharm/plugins/textmate, version=242.21829.153, package=null, isBundled=true), packagePrefix=null, state=active, parents=, )
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass(ComponentManagerImpl.kt:1464)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doLoadClass$default(ComponentManagerImpl.kt:1460)
	at com.intellij.serviceContainer.ComponentManagerImpl.loadClass(ComponentManagerImpl.kt:887)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.instantiateFileTypeBean(FileTypeManagerImpl.java:519)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.mergeOrInstantiateFileTypeBean(FileTypeManagerImpl.java:483)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.instantiatePendingFileTypes(FileTypeManagerImpl.java:476)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.getRegisteredFileTypes(FileTypeManagerImpl.java:1026)
	at com.intellij.util.indexing.IndexInfrastructureVersionBase.getAllStubFileElementTypes(IndexInfrastructureVersionBase.java:171)
	at com.intellij.util.indexing.IndexInfrastructureVersionBase.stubFileElementTypeVersions(IndexInfrastructureVersionBase.java:152)
	at com.intellij.indexing.shared.platform.api.IdeSharedIndexesState.getIndexVersion(IdeSharedIndexesState.java:68)
	at com.intellij.indexing.shared.platform.api.SharedIndexInfrastructureVersion.getIdeVersion(SharedIndexInfrastructureVersion.java:68)
	at com.intellij.indexing.shared.platform.impl.SharedIndexStorageUtil.openFileBasedIndexChunks(SharedIndexStorageUtil.java:54)
	at com.intellij.indexing.shared.platform.impl.SharedIndexChunkConfigurationImpl.openFileBasedIndexChunk(SharedIndexChunkConfigurationImpl.java:657)
	at com.intellij.indexing.shared.platform.impl.SharedIndexChunkConfigurationImpl.registerChunk(SharedIndexChunkConfigurationImpl.java:618)
	at com.intellij.indexing.shared.platform.impl.SharedIndexChunkConfigurationImpl.openChunkForProject(SharedIndexChunkConfigurationImpl.java:484)
	at com.intellij.indexing.shared.platform.impl.SharedIndexChunkConfigurationImpl.attachExistingChunk(SharedIndexChunkConfigurationImpl.java:456)
	at com.intellij.indexing.shared.platform.impl.BundledSharedIndexPostStartupActivity$execute$2.invoke(SharedIndexZipBundledStorage.kt:184)
	at com.intellij.indexing.shared.platform.impl.BundledSharedIndexPostStartupActivity$execute$2.invoke(SharedIndexZipBundledStorage.kt:182)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContextInner(coroutines.kt:339)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invokeSuspend(coroutines.kt:232)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:261)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContext(coroutines.kt:231)
	at com.intellij.indexing.shared.platform.impl.BundledSharedIndexPostStartupActivity.execute(SharedIndexZipBundledStorage.kt:182)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:496)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
2024-09-17 22:01:53,628 [   1045] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - PyCharm 2024.2.1  Build #PC-242.21829.153
2024-09-17 22:01:53,628 [   1045] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - JDK: 21.0.3; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-09-17 22:01:53,628 [   1045] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - OS: Linux
2024-09-17 22:01:53,629 [   1046] SEVERE - #c.i.o.f.i.FileTypeManagerImpl - Last Action:
Formatting /home/developer/azul/src/humancellatlas/data/metadata/lookup.py...OK
Formatting /home/developer/azul/src/humancellatlas/data/metadata/api.py...OK
Formatting /home/developer/azul/src/humancellatlas/data/metadata/age_range.py...OK
Formatting /home/developer/azul/src/humancellatlas/data/metadata/datetime.py...OK
Formatting /home/developer/azul/src/humancellatlas/data/metadata/helpers/staging_area.py...OK
Formatting /home/developer/azul/src/humancellatlas/data/metadata/helpers/json.py...OK
Formatting /home/developer/azul/src/humancellatlas/data/metadata/helpers/schema_validation.py...OK
Formatting /home/developer/azul/src/azul/doctests.py...OK
Formatting /home/developer/azul/src/azul/uuids.py...OK
Formatting /home/developer/azul/src/azul/auth.py...OK
Formatting /home/developer/azul/src/azul/time.py...OK
Formatting /home/developer/azul/src/azul/azulclient.py...OK
Formatting /home/developer/azul/src/azul/logging.py...OK
Formatting /home/developer/azul/src/azul/files.py...OK
Formatting /home/developer/azul/src/azul/enums.py...OK
Formatting /home/developer/azul/src/azul/queues.py...OK
Formatting /home/developer/azul/src/azul/terraform.py...OK
Formatting /home/developer/azul/src/azul/deployment.py...OK
Formatting /home/developer/azul/src/azul/oauth2.py...OK
Formatting /home/developer/azul/src/azul/plugins/repository/tdr_anvil/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/repository/tdr_hca/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/repository/canned/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/repository/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/repository/dss/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/repository/tdr.py...OK
Formatting /home/developer/azul/src/azul/plugins/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/bundle.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/service/aggregation.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/service/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/service/response.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/service/contributor_matrices.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/service/filter.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/indexer/aggregate.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/indexer/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/hca/indexer/transform.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/bundle.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/service/aggregation.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/service/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/service/response.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/service/filter.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/indexer/aggregate.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/indexer/__init__.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/indexer/transform.py...OK
Formatting /home/developer/azul/src/azul/plugins/metadata/anvil/schema.py...OK
Formatting /home/developer/azul/src/azul/health.py...OK
Formatting /home/developer/azul/src/azul/bigquery.py...OK
Formatting /home/developer/azul/src/azul/__init__.py...OK
Formatting /home/developer/azul/src/azul/types.py...OK
Formatting /home/developer/azul/src/azul/lambda_layer.py...OK
Formatting /home/developer/azul/src/azul/openapi/params.py...OK
Formatting /home/developer/azul/src/azul/openapi/responses.py...OK
Formatting /home/developer/azul/src/azul/openapi/__init__.py...OK
Formatting /home/developer/azul/src/azul/openapi/spec.py...OK
Formatting /home/developer/azul/src/azul/openapi/schema.py...OK
Formatting /home/developer/azul/src/azul/compliance/fedramp_inventory_service.py...OK
Formatting /home/developer/azul/src/azul/compliance/__init__.py...OK
Formatting /home/developer/azul/src/azul/docker.py...OK
Formatting /home/developer/azul/src/azul/terra.py...OK
Formatting /home/developer/azul/src/azul/threads.py...OK
Formatting /home/developer/azul/src/azul/changelog.py...OK
Formatting /home/developer/azul/src/azul/caching.py...OK
Formatting /home/developer/azul/src/azul/collections.py...OK
Formatting /home/developer/azul/src/azul/http.py...OK
Formatting /home/developer/azul/src/azul/chalice.py...OK
Formatting /home/developer/azul/src/azul/iterators.py...OK
Formatting /home/developer/azul/src/azul/es.py...OK
Formatting /home/developer/azul/src/azul/template.py...OK
Formatting /home/developer/azul/src/azul/args.py...OK
Formatting /home/developer/azul/src/azul/vendored/frozendict/__init__.py...OK
Formatting /home/developer/azul/src/azul/vendored/__init__.py...OK
Formatting /home/developer/azul/src/azul/exceptions.py...OK
Formatting /home/developer/azul/src/azul/bigquery_reservation.py...OK
Formatting /home/developer/azul/src/azul/modules.py...OK
Formatting /home/developer/azul/src/azul/json_freeze.py...OK
Formatting /home/developer/azul/src/azul/bytes.py...OK
Formatting /home/developer/azul/src/azul/service/elasticsearch_service.py...OK
Formatting /home/developer/azul/src/azul/service/catalog_controller.py...OK
Formatting /home/developer/azul/src/azul/service/source_service.py...OK
Formatting /home/developer/azul/src/azul/service/repository_controller.py...OK
Formatting /home/developer/azul/src/azul/service/repository_service.py...OK
Formatting /home/developer/azul/src/azul/service/async_manifest_service.py...OK
Formatting /home/developer/azul/src/azul/service/__init__.py...OK
Formatting /home/developer/azul/src/azul/service/drs_controller.py...OK
Formatting /home/developer/azul/src/azul/service/source_controller.py...OK
Formatting /home/developer/azul/src/azul/service/buffer.py...OK
Formatting /home/developer/azul/src/azul/service/lambda_iam_policy.py...OK
Formatting /home/developer/azul/src/azul/service/manifest_service.py...OK
Formatting /home/developer/azul/src/azul/service/storage_service.py...OK
Formatting /home/developer/azul/src/azul/service/avro_pfb.py...OK
Formatting /home/developer/azul/src/azul/service/manifest_controller.py...OK
Formatting /home/developer/azul/src/azul/hmac.py...OK
Formatting /home/developer/azul/src/azul/attrs.py...OK
Formatting /home/developer/azul/src/azul/objects.py...OK
Formatting /home/developer/azul/src/azul/version_service.py...OK
Formatting /home/developer/azul/src/azul/portal_service.py...OK
Formatting /home/developer/azul/src/azul/dss.py...OK
Formatting /home/developer/azul/src/azul/json.py...OK
Formatting /home/developer/azul/src/azul/__main__.py...OK
Formatting /home/developer/azul/src/azul/indexer/aggregate.py...OK
Formatting /home/developer/azul/src/azul/indexer/document_service.py...OK
Formatting /home/developer/azul/src/azul/indexer/index_service.py...OK
Formatting /home/developer/azul/src/azul/indexer/__init__.py...OK
Formatting /home/developer/azul/src/azul/indexer/log_forwarding_service.py...OK
Formatting /home/developer/azul/src/azul/indexer/transform.py...OK
Formatting /home/developer/azul/src/azul/indexer/document.py...OK
Formatting /home/developer/azul/src/azul/indexer/log_forwarding_controller.py...OK
Formatting /home/developer/azul/src/azul/indexer/lambda_iam_policy.py...OK
Formatting /home/developer/azul/src/azul/indexer/index_controller.py...OK
Formatting /home/developer/azul/src/azul/drs.py...OK
Formatting /home/developer/azul/src/azul/strings.py...OK
Formatting /home/developer/azul/src/azul/lambdas.py...OK
Formatting /home/developer/azul/scripts/manage_cluster_settings.py...OK
Formatting /home/developer/azul/scripts/check_branch.py...OK
Formatting /home/developer/azul/scripts/envhook.py...OK
Formatting /home/developer/azul/scripts/delete_bundle.py...OK
Formatting /home/developer/azul/scripts/confirm_sns_subscription.py...OK
Formatting /home/developer/azul/scripts/list_unit_tests.py...OK
Formatting /home/developer/azul/scripts/post_deploy_tdr.py...OK
Formatting /home/developer/azul/scripts/provision_credentials.py...OK
Formatting /home/developer/azul/scripts/reset_lambda_role.py...OK
Formatting /home/developer/azul/scripts/manage_queues.py...OK
Formatting /home/developer/azul/scripts/locust/service.py...OK
Formatting /home/developer/azul/scripts/import_default_vpc.py...OK
Formatting /home/developer/azul/scripts/reset_portal_db.py...OK
Formatting /home/developer/azul/scripts/export_inspector_findings.py...OK
Formatting /home/developer/azul/scripts/convert_environment.py...OK
Formatting /home/developer/azul/scripts/sell_unused_slots.py...OK
Formatting /home/developer/azul/scripts/generate_openapi_document.py...OK
Formatting /home/developer/azul/scripts/export_environment.py...OK
Formatting /home/developer/azul/scripts/request_flooder.py...OK
Formatting /home/developer/azul/scripts/compliance/fedramp_inventory.py...OK
Formatting /home/developer/azul/scripts/securityhub2csv.py...OK
Formatting /home/developer/azul/scripts/github_status_check.py...OK
Formatting /home/developer/azul/scripts/can_bundle.py...OK
Formatting /home/developer/azul/scripts/fetch_requirement_licences.py...OK
Formatting /home/developer/azul/scripts/stage_layer.py...OK
Formatting /home/developer/azul/scripts/import_api_gateway_stage.py...OK
Formatting /home/developer/azul/scripts/update_subgraph_counts.py...OK
Formatting /home/developer/azul/scripts/download_anvil_schema.py...OK
Formatting /home/developer/azul/scripts/manage_lambdas.py...OK
Formatting /home/developer/azul/scripts/manage_images.py...OK
Formatting /home/developer/azul/scripts/rename_resources.py...OK
Formatting /home/developer/azul/scripts/generate_environment_boot.py...OK
Formatting /home/developer/azul/scripts/reindex.py...OK
Formatting /home/developer/azul/scripts/resolve_container_path.py...OK
Formatting /home/developer/azul/scripts/kibana_proxy.py...OK
Formatting /home/developer/azul/scripts/elasticsearch_nodes.py...OK
Formatting /home/developer/azul/scripts/manage_requirements.py...OK
Formatting /home/developer/azul/scripts/terraform_schema.py...OK
Formatting /home/developer/azul/scripts/create_gitlab_snapshot.py...OK
Formatting /home/developer/azul/scripts/manage_portals.py...OK
Formatting /home/developer/azul/test/hca_metadata_api/__init__.py...OK
Formatting /home/developer/azul/test/hca_metadata_api/test.py...OK
Formatting /home/developer/azul/test/test_changelog.py...OK
Formatting /home/developer/azul/test/es_test_case.py...OK
Formatting /home/developer/azul/test/test_app_logging.py...OK
Formatting /home/developer/azul/test/dynamodb_test_case.py...OK
Formatting /home/developer/azul/test/pfb_test_case.py...OK
Formatting /home/developer/azul/test/azul_test_case.py...OK
Formatting /home/developer/azul/test/test_doctests.py...OK
Formatting /home/developer/azul/test/test_openapi.py...OK
Formatting /home/developer/azul/test/sqs_test_case.py...OK
Formatting /home/developer/azul/test/docker_container_test_case.py...OK
Formatting /home/developer/azul/test/app_test_case.py...OK
Formatting /home/developer/azul/test/health_check_test_case.py...OK
Formatting /home/developer/azul/test/version_table_test_case.py...OK
Formatting /home/developer/azul/test/service/test_portal_service.py...OK
Formatting /home/developer/azul/test/service/test_pagination.py...OK
Formatting /home/developer/azul/test/service/test_source_cache.py...OK
Formatting /home/developer/azul/test/service/test_response.py...OK
Formatting /home/developer/azul/test/service/test_pfb.py...OK
Formatting /home/developer/azul/test/service/test_buffer.py...OK
Formatting /home/developer/azul/test/service/test_app_logging.py...OK
Formatting /home/developer/azul/test/service/test_storage_service.py...OK
Formatting /home/developer/azul/test/service/test_repository_files.py...OK
Formatting /home/developer/azul/test/service/test_manifest_async.py...OK
Formatting /home/developer/azul/test/service/test_health_check.py...OK
Formatting /home/developer/azul/test/service/__init__.py...OK
Formatting /home/developer/azul/test/service/test_version_service.py...OK
Formatting /home/developer/azul/test/service/test_manifest.py...OK
Formatting /home/developer/azul/test/service/test_index_samples.py...OK
Formatting /home/developer/azul/test/service/test_drs.py...OK
Formatting /home/developer/azul/test/service/test_request_validation.py...OK
Formatting /home/developer/azul/test/service/test_index_projects.py...OK
Formatting /home/developer/azul/test/service/test_filters.py...OK
Formatting /home/developer/azul/test/service/test_response_anvil.py...OK
Formatting /home/developer/azul/test/service/test_cache_poisoning.py...OK
Formatting /home/developer/azul/test/service/test_request_builder.py...OK
Formatting /home/developer/azul/test/test_check_branch.py...OK
Formatting /home/developer/azul/test/test_log_forwarding.py...OK
Formatting /home/developer/azul/test/test_deployment.py...OK
Formatting /home/developer/azul/test/indexer/test_tdr.py...OK
Formatting /home/developer/azul/test/indexer/test_health_check.py...OK
Formatting /home/developer/azul/test/indexer/__init__.py...OK
Formatting /home/developer/azul/test/indexer/test_notifications.py...OK
Formatting /home/developer/azul/test/indexer/test_indexer.py...OK
Formatting /home/developer/azul/test/indexer/test_projects.py...OK
Formatting /home/developer/azul/test/indexer/test_indexer_controller.py...OK
Formatting /home/developer/azul/test/indexer/test_anvil.py...OK
Formatting /home/developer/azul/test/test_azul_flake8.py...OK
Formatting /home/developer/azul/test/integration_test.py...OK
Formatting /home/developer/azul/test/test_tagging.py...OK
Formatting /home/developer/azul/lambdas/layer/app.py...OK
Formatting /home/developer/azul/lambdas/indexer/app.py...OK
Formatting /home/developer/azul/lambdas/service/app.py...OK
Formatting /home/developer/azul/.flake8/azul_flake8.py...OK
Formatting /home/developer/azul/environment.py...OK
Formatting /home/developer/azul/deployments/abrahamsc.local/environment.py...OK
Formatting /home/developer/azul/deployments/achave.local/environment.py...OK
Formatting /home/developer/azul/deployments/anvilbox/environment.py...OK
Formatting /home/developer/azul/deployments/anvildev.browser/environment.py...OK
Formatting /home/developer/azul/deployments/anvildev.gitlab/environment.py...OK
Formatting /home/developer/azul/deployments/dev.shared/environment.py...OK
Formatting /home/developer/azul/deployments/anvildev/environment.py...OK
Formatting /home/developer/azul/deployments/anvilprod.browser/environment.py...OK
Formatting /home/developer/azul/deployments/anvilprod.gitlab/environment.py...OK
Formatting /home/developer/azul/deployments/anvilprod.shared/environment.py...OK
Formatting /home/developer/azul/deployments/anvilprod/environment.py...OK
Formatting /home/developer/azul/deployments/dev.browser/environment.py...OK
Formatting /home/developer/azul/deployments/dev.gitlab/environment.py...OK
Formatting /home/developer/azul/deployments/dev.shared/environment.py...OK
Formatting /home/developer/azul/deployments/dev/environment.py...OK
Formatting /home/developer/azul/deployments/gitlab.local/environment.py...OK
Formatting /home/developer/azul/deployments/hammerbox/environment.py...OK
Formatting /home/developer/azul/deployments/prod.gitlab/environment.py...OK
Formatting /home/developer/azul/deployments/dev.shared/environment.py...OK
Formatting /home/developer/azul/deployments/prod/environment.py...OK
Formatting /home/developer/azul/deployments/sandbox/environment.py...OK
Formatting /home/developer/azul/deployments/tempdev.browser/environment.py...OK
Formatting /home/developer/azul/deployments/tempdev.gitlab/environment.py...OK
Formatting /home/developer/azul/deployments/tempdev.shared/environment.py...OK
Formatting /home/developer/azul/deployments/tempdev/environment.py...OK
Formatting /home/developer/azul/.github/workflows/schedule.py...OK
Formatting /home/developer/azul/terraform/backend.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/common.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/step_function.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/lambda_layer.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/dynamo.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/cloudwatch.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/elasticsearch.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/api_gateway.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/providers.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/authentication.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/sqs.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/s3.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/gitlab/gitlab.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/shared/shared.tf.json.template.py...OK
Formatting /home/developer/azul/terraform/browser/browser.tf.json.template.py...OK
Formatting /home/developer/azul/lambdas/indexer/.chalice/config.json.template.py...OK
Formatting /home/developer/azul/lambdas/service/.chalice/config.json.template.py...OK
Formatting /home/developer/azul/.github/pull_request_template.md.template.py...OK

254 file(s) scanned.
254 file(s) formatted.
2024-09-17 22:02:00,003 [   7420]   WARN - #c.i.n.i.NotificationGroupManagerImpl - Cannot create notification group: NotificationGroupEP{id='TextMate Bundles', displayType=BALLOON, isLogByDefault=true, toolWindowId='null', bundle='messages.TextMateBundle', key='notification.group.textmate.bundles', notificationIds='null', pluginDescriptor=PluginDescriptor(name=TextMate Bundles, id=org.jetbrains.plugins.textmate, descriptorPath=plugin.xml, path=/opt/pycharm/plugins/textmate, version=242.21829.153, package=null, isBundled=true)}
java.util.MissingResourceException: Can't find bundle for base name messages.TextMateBundle, locale en_US
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2059)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1697)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1600)
	at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:1283)
	at com.intellij.AbstractBundle$Companion.resolveResourceBundleWithFallback(AbstractBundle.kt:85)
	at com.intellij.DynamicBundle.resolveResourceBundle(DynamicBundle.java:308)
	at com.intellij.DynamicBundle.lambda$getResourceBundle$4(DynamicBundle.java:277)
	at java.base/java.util.concurrent.ConcurrentMap.computeIfAbsent(ConcurrentMap.java:330)
	at com.intellij.DynamicBundle.getResourceBundle(DynamicBundle.java:277)
	at com.intellij.notification.impl.NotificationGroupEP.getDisplayName(NotificationGroupEP.java:117)
	at com.intellij.notification.impl.NotificationGroupManagerImplKt.registerNotificationGroup(NotificationGroupManagerImpl.kt:82)
	at com.intellij.notification.impl.NotificationGroupManagerImplKt.access$registerNotificationGroup(NotificationGroupManagerImpl.kt:1)
	at com.intellij.notification.impl.NotificationGroupManagerImplKt$computeGroups$1.invoke(NotificationGroupManagerImpl.kt:61)
	at com.intellij.notification.impl.NotificationGroupManagerImplKt$computeGroups$1.invoke(NotificationGroupManagerImpl.kt:60)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processWithPluginDescriptor(ExtensionPointImpl.kt:1086)
	at com.intellij.openapi.extensions.ExtensionPointName.processWithPluginDescriptor(ExtensionPointName.kt:143)
	at com.intellij.notification.impl.NotificationGroupManagerImplKt.computeGroups(NotificationGroupManagerImpl.kt:60)
	at com.intellij.notification.impl.NotificationGroupManagerImplKt.access$computeGroups(NotificationGroupManagerImpl.kt:1)
	at com.intellij.notification.impl.NotificationGroupManagerImpl.<init>(NotificationGroupManagerImpl.kt:16)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt$instantiate$2.invoke(instantiate.kt:45)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt$instantiate$2.invoke(instantiate.kt:42)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:313)
	at com.intellij.platform.instanceContainer.instantiation.InstantiateKt.instantiate(instantiate.kt:42)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceInitializer.kt:31)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance(ServiceInstanceInitializer.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invokeSuspend(LazyInstanceHolder.kt:162)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:163)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invokeSuspend(LazyInstanceHolder.kt:160)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startCoroutineUndispatched(Undispatched.kt:27)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:90)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:123)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:52)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.initialize(LazyInstanceHolder.kt:145)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.access$initialize(LazyInstanceHolder.kt:13)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.tryInitialize(LazyInstanceHolder.kt:135)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance(LazyInstanceHolder.kt:95)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext$suspendImpl(LazyInstanceHolder.kt:87)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext(LazyInstanceHolder.kt)
	at com.intellij.serviceContainer.ComponentManagerImplKt$doGetOrCreateInstanceBlocking$1.invokeSuspend(ComponentManagerImpl.kt:1568)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:277)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:111)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$BuildersKt__BuildersKt(Builders.kt:84)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:52)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at com.intellij.serviceContainer.ComponentManagerImplKt$runBlockingInitialization$1.invoke(ComponentManagerImpl.kt:1686)
	at com.intellij.serviceContainer.ComponentManagerImplKt$runBlockingInitialization$1.invoke(ComponentManagerImpl.kt:1677)
	at com.intellij.openapi.progress.ContextKt.prepareThreadContext(context.kt:85)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization(ComponentManagerImpl.kt:1677)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doGetOrCreateInstanceBlocking(ComponentManagerImpl.kt:1567)
	at com.intellij.serviceContainer.ComponentManagerImplKt.getOrCreateInstanceBlocking(ComponentManagerImpl.kt:1562)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:746)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:690)
	at com.intellij.notification.NotificationGroupManager.getInstance(NotificationGroupManager.java:19)
	at com.intellij.platform.execution.dashboard.RunDashboardCheckerActivity$RunDashboardListener.createNotification(RunDashboardCheckerActivity.kt:119)
	at com.intellij.platform.execution.dashboard.RunDashboardCheckerActivity$RunDashboardListener.isEnabled(RunDashboardCheckerActivity.kt:69)
	at com.intellij.platform.execution.dashboard.RunDashboardCheckerActivity.execute(RunDashboardCheckerActivity.kt:43)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:496)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
2024-09-17 22:02:00,341 [   7758] SEVERE - #c.i.o.v.n.RefreshQueue - null
org.jetbrains.concurrency.AsyncPromise$Companion$CANCELED$1
2024-09-17 22:02:00,341 [   7758] SEVERE - #c.i.o.v.n.RefreshQueue - PyCharm 2024.2.1  Build #PC-242.21829.153
2024-09-17 22:02:00,341 [   7758] SEVERE - #c.i.o.v.n.RefreshQueue - JDK: 21.0.3; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-09-17 22:02:00,341 [   7758] SEVERE - #c.i.o.v.n.RefreshQueue - OS: Linux
```

</details> 


<details>
<summary>Initially, the command return a non-zero exit status while still succeeding in formating the files. Then it ran successfully with warnings (as in the details above).
 </summary>

```
Formatting /home/developer/azul/lambdas/service/.chalice/config.json.template.py...OK
Formatting /home/developer/azul/.github/pull_request_template.md.template.py...OK

254 file(s) scanned.
254 file(s) formatted.

**Start Failed**

Internal error

java.lang.RuntimeException: Cannot write /tmp/PyCharmCore.format.75c34d84-b8c2-41d7-bd66-4514688e1b82.tmp/.idea/.idea/workspace.xml
	at com.intellij.configurationStore.FileBasedStorageKt.writeFile(FileBasedStorage.kt:348)
	at com.intellij.configurationStore.FileBasedStorage$FileSaveSessionProducer.saveLocally(FileBasedStorage.kt:117)
	at com.intellij.configurationStore.XmlElementStorage$XmlElementStorageSaveSessionProducer$XmlSaveSession.doSave(XmlElementStorage.kt:282)
	at com.intellij.configurationStore.XmlElementStorage$XmlElementStorageSaveSessionProducer$XmlSaveSession.access$doSave(XmlElementStorage.kt:253)
	at com.intellij.configurationStore.XmlElementStorage$XmlElementStorageSaveSessionProducer$XmlSaveSession$save$2.invoke(XmlElementStorage.kt:258)
	at com.intellij.configurationStore.XmlElementStorage$XmlElementStorageSaveSessionProducer$XmlSaveSession$save$2.invoke(XmlElementStorage.kt:258)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContextInner(coroutines.kt:339)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invokeSuspend(coroutines.kt:232)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:261)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContext(coroutines.kt:231)
	at com.intellij.configurationStore.XmlElementStorage$XmlElementStorageSaveSessionProducer$XmlSaveSession.save(XmlElementStorage.kt:258)
	at com.intellij.configurationStore.SaveSessionProducerManager.saveSessions(SaveSessionProducerManager.kt:70)
	at com.intellij.configurationStore.ProjectSaveSessionProducerManager.saveAndValidate(ProjectSaveSessionProducerManager.kt:15)
	at com.intellij.configurationStore.ProjectStoreImpl$doSave$2$1.invokeSuspend(ProjectStoreImpl.kt:376)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
Caused by: java.nio.file.FileSystemException: /tmp/PyCharmCore.format.75c34d84-b8c2-41d7-bd66-4514688e1b82.tmp/.idea/.idea: No space left on device
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:100)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:438)
	at java.base/java.nio.file.Files.createDirectory(Files.java:699)
	at com.intellij.openapi.util.io.NioFiles.tryCreateDirectory(NioFiles.java:96)
	at com.intellij.openapi.util.io.NioFiles.createDirectories(NioFiles.java:89)
	at com.intellij.openapi.util.io.NioFiles.createParentDirectories(NioFiles.java:111)
	at com.intellij.util.io.PathKt.createParentDirectories(path.kt:27)
	at com.intellij.util.io.PathKt.safeOutputStream(path.kt:41)
	at com.intellij.configurationStore.DataWriter.writeTo(DataWriter.kt:24)
	at com.intellij.configurationStore.FileBasedStorageKt.writeFile(FileBasedStorage.kt:342)
	... 22 more

-----
JRE: 21.0.3+13-b509.11 aarch64 (JetBrains s.r.o.)
/opt/pycharm/jbr
2024-09-17 21:58:23,708 [   7877]   WARN - java.util.prefs - Couldn't flush user prefs: java.util.prefs.BackingStoreException: /home/developer/.java/.userPrefs/jetbrains create failed.
make: *** [format] Error 3
```
</details>


```
❯ docker scout quickview localhost:5000/docker.io/ucscgi/azul-pycharm:2024.2.1-34
    i New version 1.13.0 available (installed version is 1.11.0) at https://github.com/docker/scout-cli
    ✓ Image stored for indexing
    ✓ Indexed 380 packages
    ✓ Provenance obtained from attestation

  Target             │  localhost:5000/docker.io/ucscgi/azul-pycharm:2024.2.1-34  │    3C     0H     1M    67L   
    digest           │  0c9f88af5dcc                                              │                              
  Base image         │  debian:bookworm-20240904                                  │    0C     0H     0M    23L   
  Updated base image │  debian:stable-20240904-slim                               │    0C     0H     0M    23L   
                     │                                                            │                              

What's next:
    View vulnerabilities → docker scout cves localhost:5000/docker.io/ucscgi/azul-pycharm:2024.2.1-34
    View base image update recommendations → docker scout recommendations localhost:5000/docker.io/ucscgi/azul-pycharm:2024.2.1-34
    Include policy results in your quickview by supplying an organization → docker scout quickview localhost:5000/docker.io/ucscgi/azul-pycharm:2024.2.1-34 --org <organization>
```

--- a/docker_images.json
+++ b/docker_images.json
@@ -31,18 +31,18 @@
             }
         }
     },
-    "docker.io/ucscgi/azul-pycharm:2024.2.1-33": {
-        "digest": "sha256:0e070d2e887c27566f8050ada4b55cf07da40e1a2e3cd0cc03427895d1c0cd48",
-        "mirror_digest": "sha256:9ec3b5d31c7b6f5168d1ed675d53a4b36eab19806c3a0eeef836b9e8e2f3f433",
+    "docker.io/ucscgi/azul-pycharm:2024.2.1-34": {
+        "digest": "sha256:b726e8599e55a088c2833f4bb6469479b6da67b9f2898112f1b133465165b73d",
+        "mirror_digest": "sha256:b15ea2a6709412a32fd832ab21d8d9ea1a1d9cfa7ea0ca4b1435a8246a6f5d61",
         "parts": {
             "linux/amd64": {
-                "digest": "sha256:66cadedd5ce726648a6c8d5fac5dbbf5264ec94baca2607a075ac5715431c095",
-                "id": "sha256:4866a88c7767960fdf74b2a788baad5467fe7555d27673058d0d4fde08a606c5",
+                "digest": "sha256:81b9a5410f8666ede4c8ba50465a110247a2f2d2d0c6424008205b56cdb104f7",
+                "id": "sha256:fd9c4a019f50096b8019f02907768a385db2c70f639161c27ce47a3c548163aa",
                 "platform": "linux/amd64"
             },
             "linux/arm64": {
-                "digest": "sha256:f21799f827eb526166c417c10526d29ee30fc34ff48f354cdd6957c4c3296940",
-                "id": "sha256:cc7d6fda770ae623e69daa8c1fb856c67b88aadbc497025666998b7da3371fe6",
+                "digest": "sha256:2cf0b704ecb5e7c04dbc7335d3982a4a8fae8ed66b84a4c81ab08b7b304b3e7b",
+                "id": "sha256:fbdf3c1a205041f05b9199df6d1cf98c490592e9dd2afe6fde27cc0642bc7f66",
                 "platform": "linux/arm64"
             }
         }

--- a/environment.py
+++ b/environment.py
@@ -282,7 +282,7 @@ def env() -> Mapping[str, Optional[str]]:
                 'url': 'https://hub.docker.com/_/python',
             },
             'pycharm': {
-                'ref': 'docker.io/ucscgi/azul-pycharm:2024.2.1-33',
+                'ref': 'docker.io/ucscgi/azul-pycharm:2024.2.1-34',
                 'url': 'https://hub.docker.com/repository/docker/ucscgi/azul-pycharm',
                 'is_custom': True
             },

